ui/apps/everest/.e2e/release/init-deploy.e2e.ts

```diff
@@ -0,0 +1,202 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { expect, test } from '@playwright/test';
+import {
+  deleteDbCluster,
+  suspendDbCluster,
+  resumeDbCluster,
+  restartDbCluster,
+} from '../utils/db-clusters-list';
+import { getTokenFromLocalStorage } from '../utils/localStorage';
+import { getClusterDetailedInfo } from '../utils/storage-class';
+import {
+  moveForward,
+  submitWizard,
+  populateBasicInformation,
+  populateResources,
+  populateAdvancedConfig,
+  populateMonitoringModalForm,
+} from '../utils/db-wizard';
+import { EVEREST_CI_NAMESPACES } from '../constants';
+import { waitForStatus, waitForDelete } from '../utils/table';
+import { checkError } from '../utils/generic';
+import { deleteMonitoringInstance } from '../utils/monitoring-instance';
+
+const { MONITORING_URL, MONITORING_USER, MONITORING_PASSWORD } = process.env;
+let token: string;
+
+test.describe.configure({ retries: 0 });
+
+[
+  { db: 'psmdb', size: 1 },
+  { db: 'psmdb', size: 3 },
+  { db: 'pxc', size: 1 },
+  { db: 'pxc', size: 3 },
+  { db: 'postgresql', size: 1 },
+  { db: 'postgresql', size: 3 },
+].forEach(({ db, size }) => {
+  test.describe(
+    'Init deploy workflow',
+    {
+      // Create/stop/resume/delete cluster in different configurations
+      tag: '@release',
+    },
+    () => {
+      const clusterName = `${db}-${size}-deploy`;
+
+      let storageClasses = [];
+      let namespace = EVEREST_CI_NAMESPACES.EVEREST_UI;
+      let monitoringName = `${db}-${size}-pmm`;
+
+      test.beforeAll(async ({ request }) => {
+        token = await getTokenFromLocalStorage();
+
+        const { storageClassNames = [] } = await getClusterDetailedInfo(
+          token,
+          request
+        );
+        storageClasses = storageClassNames;
+      });
+
+      test(`Cluster creation with ${db} and size ${size}`, async ({
+        page,
+        request,
+      }) => {
+        test.setTimeout(size * 120000);
```

Comment:
I was going through this part and noticed that you made the timeout dynamic by calculating it as size * 120,000 milliseconds. So, for a cluster of size 1, the timeout would be 120,000 milliseconds (or 2 minutes).

I'm curious...what would happen if the timeout ends up being too short in one of those test environments? Should we display an error or maybe a warning?

Also, adding a comment in the code explaining why "120000 milliseconds per size" was necessary could help to understand the reason for using `120000`. Something like:

```
// Timeout is scaled based on the cluster size to account for the additional resources needed.
test.setTimeout(size * 120000);
```

Reply:
Thank you, I have added your comment. If the test fails you will get a message that the timeout was exceeded and in which place.